lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -91,12 +101,58 @@ def last_committed_at
     def analyze_last_committed_at(targets)
       trace_writer.message "Analyzing last commit time..." do
         Parallel.each(targets, in_threads: 8) do |target|
-          stdout, _ = capture3!("git", "log", "-1", "--format=format:%aI", "--", target, trace_stdout: false, trace_command_line: false)
+          stdout, _ = git("log", "-1", "--format=format:%aI", "--", target)
           last_committed_at[target] = stdout
         end
       end
     end
 
+    def code_churn
+      @code_churn ||= {}
+    end
+
+    def analyze_code_churn
+      trace_writer.message "Analyzing code churn..." do
+        count_by_num, commits_by_num = commits_within("-n", "100")
```

Comment:
[suggestion] The long flag name seems more readable to me:

```suggestion
        count_by_num, commits_by_num = commits_within("--max-count", "100")
```

Reply:
Fixed. commit ffb52d945ee8c9cc9805d152894695a4479e78d1